howto.md

```diff
@@ -1,14 +1,31 @@
 ---
-title: How to use them
+title: How to Run a Delta Chat Bot
 ---
 
-# How to use
+# How to Run a Delta Chat Bot
+
+To run a bot, you don't need to be a programmer. 
+You don't even need to own or rent a server.
+A bot runs on any computer.
```

Comment:
.. or even on an old smartphone or raspberry pi...

Reply:
No need to repeat ourselves.